core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -492,18 +582,25 @@ public CheckpointName getCheckpointName() {
     return CheckpointName.META_MASTER;
   }
 
+  @Override
+  public String getClusterID() {
+    return mState.getClusterID();
+  }
+
   @Override
   public Iterator<Journal.JournalEntry> getJournalEntryIterator() {
-    return mPathProperties.getJournalEntryIterator();
+    return com.google.common.collect.Iterators.concat(mPathProperties.getJournalEntryIterator(),
+        mState.getJournalEntryIterator());
   }
 
   @Override
   public boolean processJournalEntry(Journal.JournalEntry entry) {
-    return mPathProperties.processJournalEntry(entry);
+    return mState.processJournalEntry(entry) || mPathProperties.processJournalEntry(entry);
```

Comment:
is this change related to this PR?

Reply:
yes, its to process `clusterInfo` in `mState`